extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -525,6 +525,37 @@ describes.realWin(
         });
       });
 
+      it('should contain aanf if have no-fill attribute', () => {
```

Comment:
Let's make the testing a bit more exhaustive.  I suggest a parameterized test pattern.  Something like the following but add more cases.

[{noFill: 'true'}, {noFill: 'ignore', notPresent: true}].forEach(({noFill, notPresent}) => {
   it (notPresent ? `should not contain aanf for ${noFill}` : `should have aanf equal to ${noFill}`, () => {
     // create sticky ad and set no fill value
     return impl.getAdUrl().then((url) => {
        if (notPresent) expect(url).to.not.match(/(\?|&)aanf=/);
        else expect(url).to.match(new RegExp(`\?|&)aanf=${noFill}`(&|$)));
     };
}
});

Reply:
Done. Thanks for the suggestion :)